server/sonar-web/src/main/js/components/controls/SearchSelect.js

```diff
@@ -87,19 +87,17 @@ export default class SearchSelect extends React.PureComponent {
     );
   };
 
-  handleBlur = () => {
-    this.setState({ options: [], query: '' });
-  };
-
   handleChange = (option /*: Option */) => {
     this.props.onSelect(option.value);
   };
 
-  handleInputChange = (query /*: string */ = '') => {
+  handleInputChange = (query /*: string */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)